github4s/src/test/scala/github4s/integration/ReposSpec.scala

```diff
@@ -471,6 +472,23 @@ trait ReposSpec extends BaseIntegrationSpec {
     response.statusCode shouldBe okStatusCode
   }
 
+  it should "successfully return results when a valid repo is provided using <owner>/<name> syntax" taggedAs Integration in {
+    val response = clientResource
+      .use { client =>
+        Github[IO](client, accessToken).repos
+          .searchRepos(Seq(validRepoOwner, validRepoName).mkString("/"), Nil)
```

Comment:
:art: 
```suggestion
          .searchRepos(s"$validRepoOwner/$validRepoName", Nil)
```

Reply:
https://github.com/47degrees/github4s/pull/680/commits/ae3310055d277a389336a3a6944eb4846064842f

@BenFradet I added removal of the import of `Seq` as well since it's no longer needed.